packages/landingpage/utils/context/UiContext.tsx

```diff
@@ -0,0 +1,19 @@
+import { createContext, ReactNode, useState } from 'react';
+
+export interface UiContextType {
+  hideFooter: boolean;
+  setHideFooter: (value: boolean) => void;
+}
+
+export const UiContext = createContext<UiContextType | null>(null);
+
+const UiContextProvider = ({ children }: { children: ReactNode }) => {
+  const [hideFooter, setHideFooter] = useState<boolean>(false);
```

Comment:
Super minor nitpicky thing: For booleans, it is often helpful to use a prefix that denotes a binary condition. For example, `is`, `has`, `can`, `should` etc. f.e. `isFooterHidden`